Makefile

```diff
@@ -17,8 +17,6 @@ help:
 	@echo "make test              Run the unit tests"
 	@echo "make functests         Run the functional tests"
 	@echo "make bddtests          Run the gherkin tests"
-	@echo "make docstrings        View all the docstrings locally as HTML"
-	@echo "make checkdocstrings   Crash if building the docstrings fails"
```

Comment:
PyLint's `docparams` extension finds missing or incorrect `:param`'s, `:raise`'s etc in docstrings. Just use that instead of using Sphinx to check the docstrings for us.

Reply:
+100 to the above